woocommerce/woocommerce-gutenberg-products-block
# Move `Combobox` to components package

## What

Moves the `Combobox` component to the components package and updates the places it is imported:

- `CountryInput`
- `StateInput`

## Why


As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add items to your cart and go to the Checkout block.
2. Ensure the country input works correctly, select United States.
3. Ensure the state input works correctly, select any state.
4. Check out and ensure the order is placed correctly with the correct country and state.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `Combobox` component available in the `@woocommerce/blocks-components` package.
